src/main/java/me/xemor/superheroes/skills/skilldata/DamageModifierData.java

```diff
@@ -0,0 +1,102 @@
+package me.xemor.superheroes.skills.skilldata;
+
+import me.xemor.superheroes.configurationdata.comparison.SetData;
+import me.xemor.superheroes.org.jetbrains.annotations.NotNull;
+import org.bukkit.configuration.ConfigurationSection;
+import org.bukkit.entity.EntityType;
+import org.bukkit.event.entity.EntityDamageEvent;
+
+public class DamageModifierData extends SkillData {
+
+    private final SetData<EntityType> entitywhitelist;
+
+    private final SetData<EntityType> entityblacklist;
+
+    private final SetData<EntityDamageEvent.DamageCause> causeblacklist;
+
+    private final SetData<EntityDamageEvent.DamageCause> causewhitelist;
+
+
+    private final double expectedMaximumDamage;
+
+    private final double damageCap;
+
+    private final double damageMin;
+    private final int priority;
+    private final boolean incoming;
+    private final boolean outgoing;
+    private final boolean eased;
+
+    public DamageModifierData(int skill,
+                              @NotNull ConfigurationSection configurationSection) {
+        super(skill, configurationSection);
+
+        expectedMaximumDamage = configurationSection.getDouble("expectedMaxDamage", 30);
+        damageCap = configurationSection.getDouble("damageMax", 15);
+        damageMin = configurationSection.getDouble("damageMin", 0);
```

Comment:
Make your ordering of max and min with word damage consistent in the user facing variables, ie maxDamage and expectedMaxDamage or expectedDamageMax and damageMax (the second ones sound stupid so probs go with first lot)